torch/fx/experimental/Partitioner.py

```diff
@@ -0,0 +1,151 @@
+from torch.fx import Graph, GraphModule
+from torch.fx import Node
+from typing import Dict, List, Tuple
+"""a graph module may not fit into one backend.
+Partitioner class helps cut one graph into subgraphs (partitions),
+so that each partition could fit into different backends
+"""
+class Partition:
+    def __init__(self, partition_id: int) -> None:
+        self.parents: List[Partition] = []
+        self.children: List[Parttion] = []
+        """
+           One partion only has to fit into one device
```

Comment:
Not sure what this comment is saying, your partitioner isn't dealing with heterogeneous partitioning, so backend type doesn't really come into play. The docstring for Partition should probably describe the partition_id, along with parents, children, and in the future size of constants and Inputs/Outputs.